lib/api/controllers/adminController.js

```diff
@@ -102,6 +102,49 @@ function AdminController(kuzzle) {
   };
 
   /**
+   * Returns the Kuzzle configuration
+   *
+   * @param {RequestObject} requestObject
+   * @returns {Promise} promise resolved as a ResponseObject
+   */
+  this.getConfig = requestObject => {
+    var modifiedRequestObject = null;
+
+    return kuzzle.pluginsManager.trigger('data:beforeGetConfig', requestObject)
+      .then(newRequestObject => {
+        var pluginsConfig = {};
+
+        modifiedRequestObject = newRequestObject;
+
+        return kuzzle.internalEngine
```

Comment:
Hum, it's a good question.

In `kuzzle.pluginsManager.plugins` we have the configuration context of the plugin. Not each plugin require the kuzzle's config.